checker/bin/hive-checker.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+LAUNCHER=
+# If debugging is enabled propagate that through to sub-shells
+if [[ "$-" == *x* ]]; then
+  LAUNCHER="bash -x"
+fi
+CHECKER_BIN_PATH=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+USAGE="Usage: alluxio-checker.sh hive [-mode USER_MODE] [-hiveurl HIVE_URL] [-name HIVE_USER_NAME] [-password HIVE_USER_PASSWORD]
+
+Prerequisites:
+    Hive cluster and Alluxio cluster are running.
+    hive-server2 is running.
+    alluxio.master.hostname is configured in your <ALLUXIO_HOME>/conf/alluxio-site.properties.
+
+Argument:
+    -hiveurl HIVE_URL is a Hive url in the form jdbc:subprotocol:subname.
+
+Optional arguments:
+    -mode USER_MODE is one of the following value, by default the value is location.
+          dfs                    Alluxio is configured as Hive default filesytem.
+          location               Alluxio is used as a location of Hive tables other than Hive default filesystem.
+    -user HIVE_USER_NAME         is the Hive user on whose behalf the connection is being made, by default is the system username.
+    -password HIVE_USER_PASSWORD is the Hive user's password, by default is an empty string.
+
+-h  display this help."
+
+ALLUXIO_PATH=$(cd "${CHECKER_BIN_PATH}/../../"; pwd)
+
+function generate_input() {
+  [ -f "./IntegrationReport.txt" ] && rm "./IntegrationReport.txt"
+
+  if [[ "${HIVE_USER_MODE}" == "location" ]]; then
+    echo "Generating Alluxio test folder."
+    echo "Running <ALLUXIO_HOME>/bin/alluxio fs mkdir /alluxioTestFolder"
+    output=$(${LAUNCHER} "${ALLUXIO_BIN_PATH}" fs mkdir /alluxioTestFolder)
+    echo $output
```

Comment:
would it make sense to only print the output in case of error?

Reply:
I change the code to print the output in case of error, but folder already exist is also an error.
When permission denied, I want users to create the test folder by themselves so that checker only exits when the error is permission denied.

Reply:
could we delete it first so that it won't already exist?

Reply:
When Alluxio permission denied, I want users to create folder and change permission manually (like ssh into a remote computer--Alluxio master node) and then rerun the checker. 

To support this case, I do not delete folder first.
A little tricky here.

Reply:
what if we use the `bin/alluxio fs test` command to check whether it exists, and only create it if it doesn't already exist?

Reply:
Sounds Good! Fixed, thanks